lib/galaxy/config/__init__.py

```diff
@@ -615,11 +615,7 @@ def _process_config(self, kwargs):
             log.warning("preserve_python_environment set to unknown value [%s], defaulting to legacy_only")
             self.preserve_python_environment = "legacy_only"
         self.nodejs_path = kwargs.get("nodejs_path")
-        # Older default container cache path, I don't think anyone is using it anymore and it wasn't documented - we
-        # should probably drop the backward compatiblity to save the path check.
         self.container_image_cache_path = self._in_data_dir(kwargs.get("container_image_cache_path", "container_images"))
```

Comment:
I think that's what actually is the present default (at least it's the directory I have in my dev repo).

```suggestion
        self.container_image_cache_path = self._in_data_dir(kwargs.get("container_image_cache_path", "container_cache"))
```